src/openrct2/localisation/StringIds.h

```diff
@@ -3958,6 +3958,8 @@ enum
 
     STR_DESYNC_REPORT = 6318,
 
+    STR_PEEP_DEBUG_SPRITE_INDEX = 6322,
```

Comment:
Why is this non-contiguous?

Reply:
Likely popped up during the rebase.

@nexgenration Could you move the string id so they are ordered ascending once more? You don't need to rebase, just add a new commit. We can squash it when we merge your PR.

Reply:
Strings 6319-6321 have been added in the meantime, so the OP used 6322, but didn't rebase it correctly.

Reply:
@AaronVanGeffen They _are_ ordered ascending, it's just that 6319-6321 are missing. His usage of 6322 is correct.

Reply:
Oh, odd that their IDs would be missing, then. The PR diff suggests he did rebase correctly, though — there are no deletions here.

Leave as-is, for now, then?

Reply:
Found them:

https://github.com/OpenRCT2/OpenRCT2/blob/e9549e88daa9710e7b16d5218be31c8c2d37802c/src/openrct2/localisation/StringIds.h#L3941-L3944

Reply:
not sure what you are asking. i assume you mean that it is not with the other strings of similar type in terms of which line it is on. in that case, i just fixed that for my next push.